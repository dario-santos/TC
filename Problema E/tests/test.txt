3
S -> _
S -> _
A -> b
