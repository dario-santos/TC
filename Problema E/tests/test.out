NULL(S) = True
NULL(A) = False
FIRST(S) =
FIRST(A) = b
FOLLOW(S) = #
FOLLOW(A) =
